src/components/alert/alert.tsx

```diff
@@ -270,6 +278,13 @@ export class Alert {
   /** is the alert queued */
   @State() queued = false;
 
+  private containerDiv: HTMLDivElement;
+
+  private setContainerDiv = (el): void => {
+    this.containerDiv = el;
+    this.containerDiv.addEventListener("transitionrun", this.onTransitionRun);
```

Comment:
@Elijbet does `onTransitionrun` not work on the JSX element?

Reply:
I think it would if Typescript didn't fail the build.  We couldn't figure out how to get Typescript to not complain about a property it doesn't know about on the div element.  If you know of a way to override types in JSX that might work, but I suggested this method to get around that because I couldn't figure out a way to get past that TS compile issue with the JSX method.

Reply:
Perhaps an upgrade to the types package might also resolve it?  Not sure...

Reply:
thanks! sounds good.